src/main/java/Duke.java

```diff
@@ -29,20 +30,52 @@ private static void replyUser(){
             if(input.equals("bye")) {
                 printMessage("Bye. Hope to see you again soon!");
                 printBorder();
-                return;
+                break;
             }
-            if(input.equals("list")){
-                for (int i = 0; i < tasksLists.size(); i++ ){
-                    printMessage((i+1) + ". " + tasksLists.get(i));
+            else if(input.equals("list")){
+                if (tasksList.size() == 0){
+                    printMessage("There are no tasks in your listlis!");
+                }
+                else {
+                    System.out.println("Here are the tasks in your list:");
+                    for (int i = 0; i < tasksList.size(); i++) {
+                        printMessage((i + 1) + "." + "[" + tasksList.get(i).getStatusIcon() + "] " + tasksList.get(i).getDescription());
+                    }
                 }
-                printBorder();
             }
             else{
-                tasksLists.add(input);
-                printMessage("added: " + input);
-                printBorder();
+                String[] completedTask = input.split(" ");
+                if (completedTask.length == 2 && completedTask[0].equals("done") && isNumeric(completedTask[1])){
```

Comment:
Another person suggested that I could use the "startsWith" method for strings to check. I thought it was an efficient way to do the conditional statement, you could consider using this too.